uint8 channel
uint8[] data